gensim/utils.py

```diff
@@ -1005,15 +1004,13 @@ def pyro_daemon(name, obj, random_suffix=False, ip=None, port=None, ns_conf={}):
 
 def has_pattern():
     """
-    Function to check if there is installed pattern library
+    Function which returns a flag indicating whether pattern is installed or not
     """
-    pattern = False
     try:
-        from pattern.en import parse
-        pattern = True
+        import pattern
```

Comment:
I have fixed it. During testing there was some bug in Travis's python2.7 and python3.3 initialization. Can you please check that?